src/codeql.ts

```diff
@@ -1,8 +1,10 @@
 import fs from "fs";
 import path from "path";
 
-import * as exec from "@actions/exec";
-import * as tc from "@actions/tool-cache";
+import exec from "@actions/exec";
+import tc from "@actions/tool-cache";
+
+class DatabaseUnpackingError extends Error {}
```

Comment:
In a way this is a trivial case, and probably fine as a built-in `Error` but I like to suggest throwing custom error types as a matter of course, so that future error handling is easier to extend and to allow for handling of specific exceptions without reading their message content.